en/ros/external_position_estimation.md

```diff
@@ -70,6 +82,37 @@ We call *x_{mav}*, *y_{mav}* and *z_{mav}* the coordinates that are sent through
 
 Regarding the orientation, keep the scalar part *w* of the quaternion the same and swap the vector part *x*, *y* and *z* in the same way. You can apply this trick with every system; you need to obtain a NED frame, look at your mocap output and swap axis accordingly.
 
+## Specific System Setups
+
+### OptiTrack MOCAP
+
+The following steps explain how to feed position estimates from an [OptiTrack](http://optitrack.com/systems/#robotics) system to PX4. It is assumed that the MOCAP system is calibrated. See [this video](https://www.youtube.com/watch?v=cNZaFEghTBU) for a tutorial on the calibration process.
+
+**Steps to do on the MOCAP software, Motive**
+
+* Align your robot's forward direction with the the [system +x-axis](https://v20.wiki.optitrack.com/index.php?title=Template:Coordinate_System)
+* [Define a rigid body in the Motive software](https://www.youtube.com/watch?v=1e6Qqxqe-k0). Give the robot a name that does not contain spaces, e.g. `robot1` instead of `Rigidbody 1`
+* [Enable Frame Broadacst and VRPN streaming](https://www.youtube.com/watch?v=yYRNG58zPFo)
+* Set the Up axis to be the Z axis (the default is Y)
+
+**Getting pose data into ROS**
+
+* Install the `vrpn_client_ros` package
+* You can get each rigid body pose on an individual topic by running
+```bash
+roslaunch vrpn_client_ros sample.launch server:=<mocap machine ip>
+```
+If you named the rigidbody as `robot1`, you will get a topic like `/vrpn_client_node/robot1/pose`
+
+**Relaying pose data to PX4**
+
+MAVROS provies a plugin to relay pose data published on `/mavros/vision_pose/pose` to PX4. Assuming that MAVROS is running, you just need to **remap** the pose topic that you get from MOCAP `/vrpn_client_node/<rigid_body_name>/pose` directly to `/mavros/vision_pose/pose`.
```

Comment:
MAVROS "provides"

Reply:
Also, this is applicable only to EKF2. You can use the mocap topic of MAVROS to feed `ATT_POS_MOCAP` data to PX4, as LPE is able to use this data as well.